client/src/hooks/useAnalytics.tsx

```diff
@@ -1,13 +1,13 @@
-import { init, trackPages } from 'insights-js'
+import Plausible from 'plausible-tracker'
 import { useEffect } from 'react'
 
+const { trackPageview } = Plausible()
+
 export const useAnalytics = () => {
   useEffect(() => {
-    const INSIGHTS_PROJECT_ID = process.env.REACT_APP_INSIGHTS_PROJECT_ID
-
-    if (INSIGHTS_PROJECT_ID && INSIGHTS_PROJECT_ID !== '') {
-      init(INSIGHTS_PROJECT_ID)
-      trackPages()
-    }
+    Plausible({
```

Comment:
Some questions:
1. Is the `Plausible` class meant to be initialized multiple times?
2. I see you pass `domain` here, but not in the other calls, is that correct?
3. Is it correct that we need to call plausible here, but also outside of the component?

Reply:
1. I blindly followed the docs, but i now moved the initialization to a separate file.
2. Yes, you initialize the tracker once.
3. Refactored.